### PR TITLE
Uninstall module used on PS 1.6 before using this one

### PR DESCRIPTION
**Pull request by Quetzacoalt91**

### CODE REVIEW

File: ps_shoppingcart.php [hunk 1]
```diff
@@ -142,6 +149,26 @@ public function install()
...
+    public function uninstallPrestaShop16Module()
+    {
+        if (!Module::isInstalled(self::PS_16_EQUIVALENT_MODULE)) {
+            return;
+        }
+        $oldModule = Module::getInstanceByName(self::PS_16_EQUIVALENT_MODULE);
+        if ($oldModule) {
+            // This closure calls the parent class to prevent data to be erased
+            // It allows the new module to be configured without migration
+            $parentUninstallClosure = function() {
```
Comment (PierreRambaud) [inline ps_shoppingcart.php]: 👍

**Comment by matks** [comment]
@Quetzacoalt91 Pr is ✅but has now conflicts

**Comment by matks** [comment]
QA found an issue with Upgrade

**Comment by matks** [comment]
We need to fix this before being able to release next version v2.0.3

**Comment by Progi1984** [comment]
Hello @Quetzacoalt91, thanks for your contribution, would you mind to rebase your pull request? Thanks

**Comment by Quetzacoalt91** [comment]
Here you go!

**Comment by Progi1984** [comment]
Thanks @Quetzacoalt91

**Pull request merged**